### PR TITLE
Override save! method in resume model

### PR DESCRIPTION
**Pull request by arun1595**
Fixes https://github.com/AgileVentures/MetPlus_tracker/issues/661

### CODE REVIEW

File: spec/models/resume_spec.rb [hunk 1]
```diff
@@ -81,7 +81,6 @@
     end
 
     it 'fails with invalid model and invalid file type' do
-      file = fixture_file_upload('files/Test File.zzz')
       resume = Resume.new(file: file,
```
Comment (patmbolger) [inline spec/models/resume_spec.rb]: Removing this line causes this test to fail.

Comment (arun1595) [reply spec/models/resume_spec.rb]: I am sorry, I don't understand.

Why do you remove that line?

Comment (patmbolger) [reply spec/models/resume_spec.rb]: @arun1595 - I _didn't_ remove that line.  That line was removed in your PR :)

Comment (arun1595) [reply spec/models/resume_spec.rb]: My bad. I misunderstood you were talking about line the comment is made on.

**Comment by patmbolger** [comment]
Looks good.

**Pull request merged**